client/cmd/internal/plan/compositor.go

```diff
@@ -0,0 +1,76 @@
+package plan
+
+type Compositor struct {
```

Comment:
this is a part of plan MR, it will be omitted
compositor will be removed entirely, and use plans struct instead for the operations
cc: @okysetiawan